server/helpers/loadConfig.js

```diff
@@ -43,6 +43,7 @@ const {
   ADALITE_MOCK_TX_SUMMARY_SUCCESS,
   ADALITE_TREZOR_CONNECT_URL,
   ADALITE_SUPPORT_EMAIL,
+  ADALITE_TRACKING_ID,
```

Comment:
I would name this `ADALITE_GA_TRACKING_ID` to make clear to what service it is related